solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/impl/TestRankingFeature.java

```diff
@@ -84,7 +84,9 @@ public void testRankingSolrFeature() throws Exception {
     query.remove("rq");
     query.add("rq", "{!ltr model=powdesS-model reRankDocs=4}");
 
-    assertJQ("/query" + query.toQueryString(), "/error/msg/=='java.lang.UnsupportedOperationException'");
+    assertJQ("/query" + query.toQueryString(), 
+        "/error/msg/=='org.apache.solr.ltr.util.FeatureException: " +
```

Comment:
is it possible to get the toString() of the feature that failed in here, so we can get a feature name?  Otherwise we might have 1000 features and won't know which one failed